client/components/TestPlanUpdater/TestPlanUpdaterModal.jsx

```diff
@@ -0,0 +1,555 @@
+                                                );
+                                            }}
+                                            checked={backupChecked}
+                                        />
+                                        Backup Test Plan before Update
+                                    </Form.Check.Label>
+                                </Form.Check>
+                            )}
+                            <div
+                                className={
+                                    testsToDelete?.length > 0
+                                        ? 'side-button-container'
+                                        : 'side-button-container no-backup'
+                                }
+                            >
+                                <Button
+                                    variant="secondary"
+                                    onClick={handleClose}
+                                    className="cancel-button"
+                                >
+                                    Cancel
+                                </Button>
+                                <Button
+                                    variant={
+                                        !backupChecked ? 'danger' : 'primary'
+                                    }
+                                    className="submit-button"
+                                    disabled={!canCreateNewReport}
+                                    onClick={createNewReportWithData}
+                                >
+                                    Update Test Plan
+                                </Button>
+                            </div>
+                        </div>
+                    </Modal.Footer>
+                </>
+            )}
+            {loadingSpinnerProgress.visible && (
+                <LoadingSpinner
+                    percentage={Math.floor(
+                        (loadingSpinnerProgress.numerator /
+                            loadingSpinnerProgress.denominator) *
+                            100
+                    )}
+                    className="loading-spinner"
+                />
+            )}
+            {alertCompletion.visible && (
+                <>
+                    <Modal.Body>
+                        <Alert
+                            className="completion-alert"
+                            variant={
+                                alertCompletion.success ? 'success' : 'danger'
+                            }
+                        >
+                            {alertCompletion.message}
+                        </Alert>
+                    </Modal.Body>
+                </>
+            )}
+            {!showModalData && (
+                <Modal.Footer className="test-plan-updater-footer">
+                    {(!closeButton && (
+                        <Button variant="secondary" onClick={deleteOldTestPlan}>
+                            Next
```

Comment:
Could this be red and say "Delete"? That's assuming this suggestion doesn't conflict with any specific direction you've already received about how to implement this.